classes/DepositPackage.inc.php

```diff
@@ -575,15 +578,20 @@ public function updateDepositStatus() {
 
 		// retrieve the content document
 		$result = $plnPlugin->curlGet($url);
+		if (intdiv((int) $result['status'], 100) !== 2) {
+			if ($result['status']) {
+				error_log(__('plugins.generic.pln.error.http.swordstatement', array('error' => $result['status'], 'message' => $result['error'])));
+
+				// Status 4XX means the deposit doesn't exist or isn't related to the given journal, so we restart the deposit
+				if (intdiv($result['status'], 100) === 4) {
```

Comment:
If we're trying to grab the status of a deposit, it means it was transferred to the staging server (or the deposit has a `NULL` status), therefore if it cannot be found on the staging server, it should be ok to reset.

This check probably won't be needed anymore, but it will cover the "status reset", so it needs to be kept.